aws_missing_permissions/main.yaml

```diff
@@ -0,0 +1,84 @@
+# version: <VERSION_PLACEHOLDER>
+AWSTemplateFormatVersion: 2010-09-09
+Description: Datadog AWS Integration Attach Missing Permissions
+Parameters:
+  DatadogIntegrationRole:
+    Type: String
+    Description: The name of the IAM role that has been integrated with Datadog.
+  AccountId:
+    Type: String
+    Description: The AWS account ID of the account where the Datadog integration role is located.
+  PolicyName:
+    Type: String
+    Description: The name of the inline policy to attach to the Datadog integration role.
+Resources:
+  AccountIdValidatorLambdaExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Path: "/"
+      ManagedPolicyArns:
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
```

Comment:
This is the managed policy, we also use this in our other stacks: https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AWSLambdaBasicExecutionRole.html